.travis.yml

```diff
@@ -31,8 +31,9 @@ matrix:
       env:
         - DEPENDS="cython numpy scipy matplotlib h5py nibabel cvxopt scikit_learn"
 before_install:
+    - apt-get update && apt-get install -y python-vtk
```

Comment:
I believe you'll need sudo for the apt-get.  I think you can use sudo for selected builds - but at the moment the global setting is `sudo: false` - which makes the builds faster.   If you want `python-vtk` for all builds, you can add it to the `apt` package list - e.g 

```
addons:
  apt:
    packages:
      - python-vtk
```

See the matplotlib `.travis.yml` file for an example.